### PR TITLE
Adding PX4 param file

### PR DESCRIPTION
**Pull request by ahcorde**
I would like to include the PX4 parameter file as another example because interacting with PX4 it's one of the targets of this pkg

### CODE REVIEW

File: ros2_serial_example/config/px4_serial_to_ros2_bridge_params.yaml [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+                                       # to feel sluggish.
+        dynamic_serial_mapping_ms: 0 # How many milliseconds to wait to get
+                                       # dynamic ROS2-to-serial mapping from the
+                                       # serial side.  If less than 0, dynamic
+                                       # mapping is disabled and the topics
+                                       # below will be used.  If exactly 0, wait
+                                       # forever for the other side to respond.
+                                       # If greater than 0, wait that many
+                                       # milliseconds for a response before
+                                       # failing.  If this config is >= 0, the
```
Comment (clalancette) [inline ros2_serial_example/config/px4_serial_to_ros2_bridge_params.yaml]:
```suggestion
                                       # failing.  If this config is >= 0, then
```

File: ros2_serial_example/config/px4_serial_to_ros2_bridge_params.yaml [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+                                       # Larger numbers will result in less CPU
+                                       # usage but also some latency in
+                                       # delivering data from ROS 2 to the
+                                       # serial port.  The default of 4 ms
+                                       # (250Hz) is a good compromise between
+                                       # CPU time and latency.  It is not
+                                       # recommended to set this above 100,
+                                       # since this will cause the application
+                                       # to feel sluggish.
+        dynamic_serial_mapping_ms: 0 # How many milliseconds to wait to get
```
Comment (clalancette) [inline ros2_serial_example/config/px4_serial_to_ros2_bridge_params.yaml]:
```suggestion
        dynamic_serial_mapping_ms: 0  # How many milliseconds to wait to get
```

More importantly, this is currently set to 0, meaning that we'll be using dynamic mapping (and not static mapping), but there are a whole bunch of static topic maps setup below.  That's fine for runtime (as they'll be ignored), but as an example I think it is a bit confusing.  Do you mean to use dynamic or static mapping in this example?

Comment (ahcorde) [reply ros2_serial_example/config/px4_serial_to_ros2_bridge_params.yaml]: Nope, according with comments
 
> If less than 0, dynamic mapping is disabled static mapping and the topics below will be used

I think it's well configure for static mapping

Comment (clalancette) [reply ros2_serial_example/config/px4_serial_to_ros2_bridge_params.yaml]: > > If less than 0, dynamic mapping is disabled static mapping and the topics below will be used
> 
> I think it's well configure for static mapping

So this is the point that I'm confused about.  The comment says "if *less* than zero", but this configuration is currently setting it to exactly 0, which means wait forever for a dynamic map.

Comment (ahcorde) [reply ros2_serial_example/config/px4_serial_to_ros2_bridge_params.yaml]: This line of code is the key:
https://github.com/osrf/ros2_serial_example/blob/master/ros2_serial_example/src/ros2_to_serial_bridge.cpp#L125

less or equal to zero, right?

Comment (clalancette) [reply ros2_serial_example/config/px4_serial_to_ros2_bridge_params.yaml]: Oh, I see.  OK, that is actually a bug in the code; that line of code really should be:

```
if (dynamic_serial_mapping_ms >= 0)
```

My suggestion is to change this PR to set `dynamic_mapping_ms` to -1 (which will work for both the current code and any new code), and then open a separate PR to fix that bug.

Comment (ahcorde) [reply ros2_serial_example/config/px4_serial_to_ros2_bridge_params.yaml]: Done ;) and created the PR https://github.com/osrf/ros2_serial_example/pull/74

**Comment by ahcorde** [comment]
Thanks for your support @clalancette

**Pull request merged**